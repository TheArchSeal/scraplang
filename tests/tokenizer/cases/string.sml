""
"abc"
"'"
"\'"
"\""
"\n"
"\r"
"\t"
"a\0b"
"\x61"
"Z\x5AZ"
