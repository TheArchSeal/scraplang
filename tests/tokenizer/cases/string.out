(4):1:1 "" 
(4):2:1 "abc" abc
(4):3:1 "'" '
(4):4:1 "\'" '
(4):5:1 "\"" "
(4):6:1 "\n" 

(4):7:1 "\r" 
(4):8:1 "\t" 	
(4):9:1 "a\0b" a
(4):10:1 "\x61" a
(4):11:1 "Z\x5AZ" ZZZ
